README.md

```diff
@@ -1 +1,39 @@
-# Bookstore
\ No newline at end of file
+# Bookstore
+
+"The Bookstore" is a react web app built to show collections of books, and give users access to add new books or remove existing book
+
+![Screenshot (121)](https://user-images.githubusercontent.com/68381641/171909117-5139133e-3097-4a8b-aa86-b158018e9a06.png)
+
+## Built With
+- CSS
+- React
+
+## Live Demo
+[127.0.0.1:5500/index.html](http://127.0.0.1:5500/index.html)
+## Getting Started
+To get a local copy open and running follow these simple example steps.
```

Comment:
- [ ] [OPTIONAL] Please remove this link to your live demo because it doesn't work. You can indicate that you're currently working on it instead 👍